richtext-commonmark/src/commonJvmAndroid/kotlin/com/halilibo/richtext/markdown/AstNodeConvert.kt

```diff
@@ -188,19 +189,20 @@ internal fun convert(
 internal fun Node.convert() = convert(this)
 
 @Composable
-internal actual fun parsedMarkdownAst(text: String): AstNode? {
-  val parser = remember {
+internal actual fun parsedMarkdownAst(text: String, options: MarkdownParseOptions): AstNode? {
+  val parser = remember(options) {
     Parser.builder()
       .extensions(
-        listOf(
+        listOfNotNull(
           TablesExtension.create(),
-          StrikethroughExtension.create()
+          StrikethroughExtension.create(),
+          if (options.autolink) AutolinkExtension.create() else null
         )
       )
       .build()
   }
 
-  val astRootNode by produceState<AstNode?>(null, text) {
+  val astRootNode by produceState<AstNode?>(null, text, options) {
```

Comment:
parser should be a key to this produceState since it can change according to options.

Reply:
Good catch! I had used `options` as the key for `parser` above, and for some reason re-used the same key here instead of using `parser` 🤦